### PR TITLE
prereq for CNI podman

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/cli/pages/podman.adoc [hunk 1]
```diff
@@ -44,6 +44,12 @@ To switch back to target Kubernetes sites: `skupper switch kubernetes`
 
 * The latest `skupper` CLI is installed.
 * Podman is installed, see https://podman.io/
++
+--
+By default, Podman v4 uses Netavark which works with Skupper.
+If you are using CNI, for example, if you upgrade from Podman v3, you must also install https://www.cni.dev/plugins/current/[containernetworking-plugins].
+For example, `dnf install containernetworking-plugins` or `apt-get install containernetworking-plugins`.
```
Comment (fgiorgetti) [inline modules/cli/pages/podman.adoc:51]: @pwright I have validated the package name and the one I have provided you earlier is
incorrect. The correct one is: "podman-plugins".

Comment (pwright) [reply modules/cli/pages/podman.adoc]: updated, but note that podman-plugins isn't available for a typical ubuntu install.. On the other hand latest ubuntu has podman v4

Comment (fgiorgetti) [reply modules/cli/pages/podman.adoc]: I believe that podman v4 is only available on Ubuntu through the test/unstable channels at the moment.
But maybe it is better to add a note about this package being valid for Fedora, RHEL and CentOS.

File: modules/cli/pages/podman.adoc [hunk 2]
```diff
@@ -44,6 +44,12 @@ To switch back to target Kubernetes sites: `skupper switch kubernetes`
 
 * The latest `skupper` CLI is installed.
 * Podman is installed, see https://podman.io/
++
+--
+By default, Podman v4 uses Netavark which works with Skupper.
+If you are using CNI, for example, if you upgrade from Podman v3, you must also install the `podman-plugins` package.
+For example, `dnf install podman-plugins` for RPM based distributions.
```
Comment (pwright) [inline modules/cli/pages/podman.adoc:51]: @fgiorgetti I think this is ok? btw I spun up an Ubuntu 23.04 

```

$ sudo apt install podman
$ podman version
Client:       Podman Engine
Version:      4.3.1
API Version:  4.3.1

```

**Pull request merged**